test/test-locales.js

```diff
@@ -0,0 +1,28 @@
+var fs = require('fs');
+var path = require('path');
+var assert = require('assert');
+var glob = require('glob');
+var axe = require(path.join(__dirname, '../axe'));
+
+var localeFiles = glob.sync(path.join(__dirname, '../locales/*.json'));
+
+describe('locales', function() {
+	localeFiles.forEach(function(localeFile) {
+		var localeName = path.basename(localeFile);
+		it(localeName + ' should be valid', function(done) {
+			fs.readFile(localeFile, 'utf-8', function(err, localeData) {
+				if (err) {
+					done(err);
```

Comment:
This will release Zalgo.

Reply:
I'm not sure what you mean

Reply:
https://blog.izs.me/2013/08/designing-apis-for-asynchrony

TLDR; add a `return` here to avoid calling `done` more than once.

Reply:
Better yet, just use `fs.readFileSync` :wink: